consensus/consensus-types/src/sync_info.rs

```diff
@@ -32,40 +35,40 @@ impl Display for SyncInfo {
     fn fmt(&self, f: &mut Formatter) -> std::fmt::Result {
         write!(
             f,
-            "SyncInfo[certified_round: {}, ordered_round: {}, timeout round: {}, commit_info: {}]",
+            "SyncInfo[certified_round: {}, ordered_round: {}, timeout round: {}, committed_round: {},\n hqc: {:?},\n hoc: {:?},\n hcc: {:?}]",
             self.highest_certified_round(),
             self.highest_ordered_round(),
             self.highest_timeout_round(),
-            self.highest_commit_cert().commit_info(),
+            self.highest_commit_round(),
+            self.highest_quorum_cert,
+            self.highest_ordered_cert.as_ref().map_or("None".to_string(), |cert| cert.to_string()),
+            self.highest_commit_cert,
         )
     }
 }
 
 impl SyncInfo {
     pub fn new_decoupled(
         highest_quorum_cert: QuorumCert,
-        highest_ordered_cert: QuorumCert,
-        highest_commit_cert: QuorumCert,
+        highest_ordered_cert: WrappedLedgerInfo,
+        highest_commit_cert: WrappedLedgerInfo,
         highest_2chain_timeout_cert: Option<TwoChainTimeoutCertificate>,
     ) -> Self {
         // No need to include HTC if it's lower than HQC
         let highest_2chain_timeout_cert = highest_2chain_timeout_cert
             .filter(|tc| tc.round() > highest_quorum_cert.certified_block().round());
 
-        let highest_ordered_cert =
-            Some(highest_ordered_cert).filter(|hoc| hoc != &highest_quorum_cert);
-
         Self {
             highest_quorum_cert,
-            highest_ordered_cert,
+            highest_ordered_cert: Some(highest_ordered_cert),
             highest_commit_cert,
             highest_2chain_timeout_cert,
         }
     }
 
     pub fn new(
         highest_quorum_cert: QuorumCert,
-        highest_ordered_cert: QuorumCert,
+        highest_ordered_cert: WrappedLedgerInfo,
         highest_2chain_timeout_cert: Option<TwoChainTimeoutCertificate>,
     ) -> Self {
         let highest_commit_cert = highest_ordered_cert.clone();
```

Comment:
Why is commit cert a clone of ordered cert? It does not need to have execution state? @zekun000